src/main.cpp

```diff
@@ -73,10 +76,27 @@
 #define _MAIN_CRASH_SLEEP 2
 #endif
 
+<<<<<<< Updated upstream
 OswAppSwitcher mainAppSwitcher(BUTTON_1, LONG_PRESS, true, true, &main_currentAppIndex);
 OswAppSwitcher watchFaceSwitcher(BUTTON_1, SHORT_PRESS, false, false, &main_watchFaceIndex);
 OswAppSwitcher settingsAppSwitcher(BUTTON_1, SHORT_PRESS, false, false, &main_settingsAppIndex);
 OswAppSwitcher fitnessAppSwitcher(BUTTON_1, SHORT_PRESS, false, false, &main_fitnessAppIndex);
+=======
```

Comment:
I don't think the files are organized yet.

Reply:
yes, everything is fine with the tools screen update. I'm not sure why there are other commits being added in, they can be ignored. The tools screens are templates so tools can to the tools screen of the watch, like maybe the level screen. So this is just an update to allow the tools screen to available for that.